File: Redstone/Players/GameMode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Redstone.Players
{
	public enum GameMode : byte
	{
		Survival = 0,
		Creative = 1,
		Adventure = 2,
		Spectator = 3,
		Hardcore = 0x8
	}
}

File: src/com/CopperPenguin96/Redstone/System/Types/GameMode.java
package com.CopperPenguin96.Redstone.System.Types;

public enum GameMode {
	Survival(0),
	Creative(1),
	Adventure(2),
	Hardcore(3);
	
	public int _val;
	private GameMode(int val) {
		_val = val;
	}
}
